tasks/build.yml

```diff
@@ -59,14 +59,79 @@
       become: "no"
       when:
         - not atom_compile_all_themes|bool # Don't compile themes when they are alreay compiled
-  when: ansible_os_family == "RedHat"
+  when:
+    - ansible_os_family == "RedHat"
+    - php_version|int < 74
+
+- name: "Build AtoM themes (CentOS/RH and php version >= 7.4)"
+  block:
+    - name: "Build all AtoM themes (CentOS/RH)"
+      shell: "make"
+      args:
+        chdir: "{{ item }}"
+      with_items: "{{ theme_makefiles.stdout_lines }}"
+      become: "no"
+      when:
+        - atom_compile_all_themes|bool
+    - name: "Build selected AtoM themes (CentOS/RH)"
+      shell: "{{ item.build_cmd }}"
+      args:
+        chdir: "{{ item.path }}"
+      with_items: "{{ atom_themes }}"
+      become: "no"
+      when:
+        - not atom_compile_all_themes|bool # Don't compile themes when they are alreay compiled
+  when:
+    - ansible_os_family == "RedHat"
+    - php_version|int >= 74
+
+#
+# Bootstrap 5 support
+#
+
+- name: "Prepare npm to be used with bootstrap5 themes"
+  block:
+    - name: "Update npm to latest version of its release"
+      command: "npm install -g npm@latest"
+    - name: "Get AtoM user home directory"
+      become_user: "{{ atom_user }}"
+      become: "yes"
+      shell: >
+             getent passwd {{ atom_user }}  | awk -F: '{ print $6 }'
+      changed_when: false
+      register: __atom_user_home_directory
+    - name: "Create npm directories on {{ atom_user }} home dir"
+      become: "yes"
+      file:
+        path: "{{ __atom_user_home_directory.stdout }}/{{ item }}"
+        state: "directory"
+        owner: "{{ atom_user }}"
+        group: "{{ atom_group }}"
+      loop:
+        - ".config"
+        - ".npm"
+    - name: "Clean cache, install and build with npm on AtoM site directory"
+      become: "yes"
+      become_user: "{{ atom_user }}"
+      command: "{{ item }}"
+      args:
+        chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+      environment:
+        CYPRESS_INSTALL_BINARY: 0
+      loop:
+        - "npm cache clean --force"
+        - "npm install"
+        - "npm run build"
+  when:
+    # Removing "." for Ubuntu php_version, for instance: 7.4
+    - php_version|replace('.','')|int >= 74
```

Comment:
This appears to be the only "when: -php_version ..." place you need the extra check to remove the "." (so '7.4' --> '74'), is that correct?

(Something to do with npm or otherwise only with BS5?)